multiversion/src/main/scala/multiversion/commands/LintCommand.scala

```diff
@@ -129,9 +129,25 @@ case class LintCommand(
             )
         }
         val rendered = Doc.intercalate(Doc.line, docs).render(Int.MaxValue)
-        Files.createDirectories(out.getParent())
-        Files.write(out, rendered.getBytes(StandardCharsets.UTF_8))
+        if (out.toString() == "-") {
+          writeToStdout(rendered)
+        } else {
+          writeToFile(rendered, out)
+        }
       }
+      
+  def writeToStdout(report: String): Unit = 
+    app.println(report)
```

Comment:
This will print the report to standard out as expected, but it will also be mixed in with everything else in the log.

```suggestion
  def writeToStdout(report: String): Unit = {
    app.println("# ---- begin lint result")
    app.println(report)
    app.println("# ---- end lint result")
  }
```

Adding something like that would let you locate the YAML content afterwards. Lines starting with `#` will be treated as comment in YAML, so it's should be safe if someone were to pipe the result to something else too.